src/api.ts

```diff
@@ -63,10 +63,31 @@ export function makeConsentManagerAPI({
       Promise.resolve(
         handleSetViewState(options?.viewState || 'OptOutDisclosure', auth),
       ),
-    showConsentManager: (options) =>
-      Promise.resolve(
-        handleSetViewState(options?.viewState || 'open', undefined, true),
-      ),
+    // eslint-disable-next-line require-await
+    showConsentManager: async (options) => {
+      if (options?.viewState === ViewState.TCF_EU) {
+        logger.warn(
+          'TCF_EU view state is not valid. Please configure your regime to use this view state.',
+        );
+        return;
+      }
+      if (
+        options?.viewState &&
+        !Object.values(ViewState).includes(options.viewState)
+      ) {
+        logger.warn(
```

Comment:
Can we do error instead of warn?

Reply:
sorry i missed that. just updated it to an error. i need to do the same for the warning in the tcf-ui